bookmarks.html

```diff
@@ -14,99 +14,158 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
       <h1>Bookmarks</h1>
     </header>
 
     <main>
       <ul class="card-container" role="list">
-        <li class="card">
-          <div class="question-container">
-            <h2>Question</h2>
-            <p class="question">
+        <li class="card q-card">
+          <div class="q-card__q-container">
+            <h2 class="q-card__title text-primary">Question</h2>
+            <p class="q-card__q-text">
               Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam,
               excepturi?
             </p>
           </div>
-          <input type="checkbox" id="q1" /><label class="btn-answer" for="q1">
-            <span class="show">show answer</span>
-            <span class="hide">hide answer</span></label
-          >
-          <div class="answer-container">
-            <h2>Answer</h2>
-            <p class="answer">
+
+          <input class="hidden q-card__btn-ceckbox" type="checkbox" id="q1" />
```

Comment:
Why not just `card__button` or even `toggle-button`? No need to repeat in the name that this is a checkbox.